### PR TITLE
Covid 19 tracker

### PR DESCRIPTION
**Pull request by Banstein**
- Created Initial setup with React create app and Linters
- Desktop and mobile version optimized
- Implemented basic structure for Redux store and Components, and routing
- Added countries reducer and fetching data from 'Narrativa' API
- Rendered the fetch data in the UI
- Implemented filtering cases for countries
- Implemented showing regions data per country
- Tested the redux functions and components
- Updated README.md

### CODE REVIEW

File: src/__tests__/components/CountriesContainer.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <Provider store={store}>
+          <BrowserRouter>
+            <CountriesContainer />
+          </BrowserRouter>
+        </Provider>,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```
Comment (medaminedev66) [inline src/__tests__/components/CountriesContainer.test.js:18]: - [ ] Following the project requirements, You should also use React Testing Library to simulate user interaction for some components. Check out this [link ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/testing_react_with_react_testing_library.md)to learn how to do that.

File: README.md [hunk 2]
```diff
@@ -1,44 +1,92 @@
...
+
+Give a :star: if you like this project!
+
+## Acknowledgments
+
+- Special thanks to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) for the design
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (medaminedev66) [inline README.md:92]: - [x] The MIT link is broken, kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Banstein  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional
- your code organization was really good.
- The project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly. Great job 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,44 +1,92 @@
...
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Description
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+> "covid-19-tracker" is a webapp that allows users to track and watch the statistics of covid-19 cases for each country and also the regions within the country updated evrey day.
 
-### `npm test`
+## Screenshots
```
Comment (medaminedev66) [inline README.md:10]: - [x] I recommend removing the `screenshots` title if the screenshots are not available so your Readme is documented professionally.

**Review by rotimiazeez — APPROVED** [review]
# Hi @Banstein✅, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**